src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeGeoJsonExtension.java

```diff
@@ -0,0 +1,39 @@
+package org.openstreetmap.atlas.utilities.vectortiles;
+
+import com.google.gson.JsonObject;
+
+/**
+ * TippecanoeGeoJsonExtension fills in a JsonObject called "tippecanoe" to your GeoJSON feature.
+ * Here you can specify things like minzoom and layer name for the given feature.
+ * https://github.com/mapbox/tippecanoe/blob/master/README.md#geojson-extension
```

Comment:
Maybe make that link point to a tag, or a specific commit, to make sure it does not go away.

Reply:
done